README.md

```diff
@@ -265,7 +265,7 @@ The concrete types you will encounter throughout this documentation:
 - **Pair a b** - An array with exactly two elements: `[a, b]`.
 - **Iterator** - Objects with `next`-methods which conform to the [Iterator protocol][3].
 - **Cancel** - The nullary [cancellation](#cancellation) functions returned from computations.
-- **Catchable e f** - A function `f` which may throw an exception `e`.
+- **Throwing e a b** - A function from `a` to `b` which may throw an exception `e`.
```

Comment:
```suggestion
- **Throwing e a b** - A function from `a` to `b` that may throw an exception `e`.
```

Reply:
It sounds weird when I pronounce your suggestion versus my own. I remember reading your comment regarding the use of "that" and "which" a while ago, but I can't remember where.

Reply:
Reading up on it now.

> In fact, having a client try to overrule my correction of a *which* to a *that* was one of the things that pushed me over the edge and made me start the Grammar Girl podcast.
> -- https://www.quickanddirtytips.com/education/grammar/which-versus-that-0

It seems like a very common mistake. So common that using *that* sounds weird to me.

Reply:
> I remember reading your comment regarding the use of "that" and "which" a while ago, but I can't remember where.

sanctuary-js/sanctuary-def#241